package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.3.4",
+  "version": "0.0.0-semantically-released",
```

Comment:
I think `0.0.0-semantically-released` is only for the initial release. I am trying to find the doc where I read this long time ago.